slides/tutorial-summit-networking/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Microcontroller based IoT: a hand-on course with RIOT</title>
```

Comment:
The title should be adapted to the tutorial name: "RIOT Summit - Networking Tutorial" or something like this

Reply:
Other summit tutorials have the title: `RIOT Summit - Beginner Tutorial` but maybe they should be adapted as well.

Reply:
I'll keep `RIOT Summit - Beginner Tutorial`, not sure how far we will get